osbuild/util/parsing.py

```diff
@@ -72,16 +72,24 @@ def parse_input(url: ParseResult, args: Dict) -> os.PathLike:
     return root
 
 
-def parse_location(location: str, args: Dict) -> str:
+def parse_location_into_parts(location: str, args: Dict) -> Tuple[str, str]:
```

Comment:
Is this new function tested? If not, please add a test

Reply:
It tested only as part of `parse_location`, does it make sense to add separate tests?